node/node.go

```diff
@@ -1219,6 +1219,26 @@ func (n *Node) VerifyProof(rootHash string, address string, proof [][]byte) (boo
 	return mpv.VerifyProof(rootHashBytes, key, proof)
 }
 
+// IsDataTrieMigrated returns true if the data trie for the given address is migrated
+func (n *Node) IsDataTrieMigrated(address string) (bool, error) {
+	addressBytes, err := n.coreComponents.AddressPubKeyConverter().Decode(address)
+	if err != nil {
+		return false, err
+	}
+
+	accountHandler, err := n.stateComponents.AccountsAdapterAPI().LoadAccount(addressBytes)
```

Comment:
should we use GetExistingAccount so it will error if the account does not exist?